docs/source/intervention_models/wasting_treatment/treatment_diagram.svg

```diff
@@ -1,3 +1,3 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN" "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd">
```

Comment:
I think it's a little confusing to MAM and SAM boxes into treated and untreated categories, since our modeling strategy is actually to have everyone in the box be untreated, and move them immediately into the MILD box as soon as they get treated.